network/nodenetwork/nodekeeper.go

```diff
@@ -106,12 +107,12 @@ func createOrigin(configuration configuration.HostNetwork, certificate insolar.C
 	), nil
 }
 
-func resolveAddress(configuration configuration.HostNetwork) (string, error) {
-	addr, err := net.ResolveTCPAddr("tcp", configuration.Transport.Address)
+func resolveAddress(configuration configuration.Transport) (string, error) {
```

Comment:
I think we should remove  `resolveAddress ` and `createOrigin ` from NodeNetwork, we can get origin from host network

Reply:
I will create a separate task for this refactoring